airflow/models/serialized_dag.py

```diff
@@ -351,6 +351,21 @@ def get_latest_version_hash(cls, dag_id: str, session: Session = None) -> str |
         """
         return session.query(cls.dag_hash).filter(cls.dag_id == dag_id).scalar()
 
+    @classmethod
+    @provide_session
+    def get_latest_version_hash_and_updated_datetime(
```

Comment:
Can we add a `:meta private:` so this stays undocumented? I don’t see how this needs to be public interface.

Also the other two are now entirely unused and arguably should be removed, but we can’t due to the compatibility policy. Maybe we should deprecate them, but that doesn’t need to happen in this PR.

Reply:
sure :+1: